src/components/problemCard/ProblemCard.css

```diff
@@ -0,0 +1,80 @@
+.codeutsava__section8-problems-container1 {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: row;
+  flex-wrap: wrap;
+  gap: 4rem;
+  width: 100%;
+}
+
+.codeutsava__section8-problems-container2 {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: row;
+  flex-wrap: wrap;
+  gap: 4rem;
+  width: 100%;
+  transition: display 3s ease-in-out;
+  margin-top: 4rem;
+}
+
+@keyframes fadeIn {
+  0% {
+    opacity: 0;
+  }
+  100% {
+    opacity: 1;
+  }
+}
+
+@keyframes fadeOut {
+  0% {
+    opacity: 1;
+  }
+  100% {
+    opacity: 1;
+  }
+}
+
+.codeutsava__section8-card-body {
+  width: 40%;
+}
```

Comment:
When you'll use grid, you won't have to manually set this to 40%. Remove this after making it grid.